src/app/interfaces/cloud-sync-task.interface.ts

```diff
@@ -44,7 +44,12 @@ export interface CloudSyncTask {
   create_empty_src_dirs: boolean;
 }
 
-export type CloudSyncTaskUpdate = Omit<CloudSyncTask, 'id' | 'job' | 'locked'>;
+export type CloudSyncTaskRequestPart = Omit<CloudSyncTask, 'id' | 'job' | 'locked' | 'credentials' | 'encryption_salt' | 'args' | 'filename_encryption' | 'bwlimit'>;
```

Comment:
Doesn't seem to be used anywhere else. Either incorporate it in definition of `CloudSyncTaskUpdate` (preferred) or remove `export`.